providers/angus-mail/src/test/java/com/sun/mail/util/PropUtilTest.java

```diff
@@ -179,4 +185,30 @@ public void testSystemBoolean() throws Exception {
 	System.getProperties().put("testboolean", true);
 	assertTrue(PropUtil.getBooleanSystemProperty("testboolean", false));
     }
+
+    @Test
+    public void testScheduledExecutorWriteTimeout() {
+        final String executorPropertyName = "test";
+        ScheduledExecutorService ses = new ScheduledThreadPoolExecutor(1);
```

Comment:
We should shutdown the ses after we create it just to be future proof

Reply:
fixed